pkg/rancher/rancher.go

```diff
@@ -104,7 +104,13 @@ func New(ctx context.Context, clientConfg clientcmd.ClientConfig, opts *Options)
 	}
 
 	if features.MCM.Enabled() && !features.Fleet.Enabled() {
-		return nil, fmt.Errorf("multi-cluster-management features requires feature fleet=true")
+		logrus.Info("fleet can't be turned off when MCM is enabled. Turning on fleet and turning off continuous-delivery")
+		if err := features.SetFeature(wranglerContext.Mgmt.Feature(), features.Fleet, true); err != nil {
```

Comment:
Can we just remove the fleet feature flag if this is the case? This should solve https://github.com/rancher/rancher/issues/32688.

Reply:
We can't remove this flag, since downstream cluster stilll needs the ability to disable fleet.

Reply:
The downstream cluster runs mini rancher so it needs the ability to disable fleet